### PR TITLE
Add influx CLI 2.1 release notes

### PR DESCRIPTION
**Pull request by kelseiv**
Closes https://github.com/influxdata/docs-v2/issues/2939

### CODE REVIEW

**Comment by mluu12** [comment]
@danxmoran Concerning the OSS-only changes, will they work with the existing 2.0 or will it only work with the upcoming 2.1.0? Thank you!

**Comment by danxmoran** [comment]
@mluu12 they will work with the existing 2.0

File: content/influxdb/cloud/reference/release-notes/influx-cli.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+### `influx stacks --json` output conventions
+
+The output of `influx stacks --json` previously used an UpperCamelCase naming convention for most keys.
+The command now uses lowerCamelCase consistently for all objects keys, matching the schema returned by the API.
+
+## Features
+
+- Add global `--http-debug` flag to all `influx` commands to help inspect communication with InfluxDB servers.
+- Add [`bucket-schema` commands](/influxdb/cloud/reference/cli/influx/bucket-schema/) to manage explicit measurement schemas in InfluxDB Cloud.
```
Comment (danxmoran) [inline content/influxdb/cloud/reference/release-notes/influx-cli.md:43]: Should we rearrange this line to put `(Cloud only)` or something similar at the front? To match the `(OSS only)` on other lines, so users can immediately see

File: content/influxdb/cloud/reference/release-notes/influx-cli.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+- Update [`bucket create`](/influxdb/cloud/reference/cli/influx/bucket/create/) to allow setting a schema type.
+- Update [`bucket list`](/influxdb/cloud/reference/cli/influx/bucket/list/) to display schema types.
+- (OSS only) Updates to `backup` and `restore`:
+  - Reimplement [`backup`](/influxdb/cloud/reference/cli/influx/backup/) to support downloading embedded SQL store from InfluxDB v2.1.x.
+  - Add `--compression` flag to support enabling/disabling GZIP compression of downloaded files.
```
Comment (danxmoran) [inline content/influxdb/cloud/reference/release-notes/influx-cli.md:48]: Is there a `backup.md` that this flag should be added to?

File: content/influxdb/v2.0/reference/cli/influx/org/members/add.md [hunk 3]
```diff
@@ -16,15 +16,15 @@ influx org members add [flags]
 ```
```
Comment (danxmoran) [inline content/influxdb/v2.0/reference/cli/influx/org/members/add.md:16]: I notice these reference files are still stored in a `v2.0` folder, should they be copied to a `v2.1` and edited there?

File: content/influxdb/cloud/reference/release-notes/influx-cli.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+The output of `influx stacks --json` previously used an UpperCamelCase naming convention for most keys.
+The command now uses lowerCamelCase consistently for all objects keys, matching the schema returned by the API.
+
+## Features
+
+- Add global `--http-debug` flag to all `influx` commands to help inspect communication with InfluxDB servers.
+- Add [`bucket-schema` commands](/influxdb/cloud/reference/cli/influx/bucket-schema/) to manage explicit measurement schemas in InfluxDB Cloud.
+- Update [`bucket create`](/influxdb/cloud/reference/cli/influx/bucket/create/) to allow setting a schema type.
+- Update [`bucket list`](/influxdb/cloud/reference/cli/influx/bucket/list/) to display schema types.
+- (OSS only) Updates to `backup` and `restore`:
```
Comment (sanderson) [inline content/influxdb/cloud/reference/release-notes/influx-cli.md:46]:
```suggestion
- (InfluxDB OSS only) Updates to `backup` and `restore`:
```

File: content/influxdb/cloud/reference/release-notes/influx-cli.md (hunk 2 above)
Comment (sanderson) [reply content/influxdb/cloud/reference/release-notes/influx-cli.md:48]: @mluu, @kelseiv [`content/influxdb/v2.0/reference/cli/influx/backup/_index.md`](https://github.com/influxdata/docs-v2/blob/influx-cli-2.1/content/influxdb/v2.0/reference/cli/influx/backup/_index.md)

File: content/influxdb/cloud/reference/release-notes/influx-cli.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+  - Reimplement [`backup`](/influxdb/cloud/reference/cli/influx/backup/) to support downloading embedded SQL store from InfluxDB v2.1.x.
+  - Add `--compression` flag to support enabling/disabling GZIP compression of downloaded files.
+  - Reimplement `restore` to support uploading embedded SQL store from InfluxDB v2.1.x.
+- (OSS only) Add [`--password`](/influxdb/cloud/reference/cli/influx/user/password/) flag to `user password` command to allow bypassing interactive prompt.
```
Comment (sanderson) [inline content/influxdb/cloud/reference/release-notes/influx-cli.md:50]:
```suggestion
- (InfluxDB OSS only) Add [`--password`](/influxdb/cloud/reference/cli/influx/user/password/) flag to `user password` command to allow bypassing interactive prompt.
```

File: content/influxdb/v2.0/reference/cli/influx/org/members/add.md (hunk 3 above)
Comment (sanderson) [reply content/influxdb/v2.0/reference/cli/influx/org/members/add.md:16]: @danxmoran The minor version directories right now are scoped to OSS releases. As far as we understand, the new CLI is being released immediately with a 2.1.0 tag, but the full OSS release won't happen until the end of Augusts. So for now, we're making all the CLI-specific changes in the v2.0 directory to support the new CLI release and will port everything over to 2.1 as we get closer to the full 2.1 release.

File: content/influxdb/cloud/reference/release-notes/influx-cli.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+## New repository
+
+This is the initial release of the `influx` CLI from the `influxdata/influx-cli` GitHub repository.
+
+## Breaking changes
+
+### `influx write` skip-header parsing
+
+The `write` command no longer supports `--skipHeader` as short-hand for `--skipHeader 1`.
+This change was made to simplify our CLI parser.
```
Comment (sanderson) [inline content/influxdb/cloud/reference/release-notes/influx-cli.md:22]:
```suggestion
To simplify the CLI parser, the `write` command no longer supports `--skipHeader`
as short-hand for `--skipHeader 1`.
```

File: content/influxdb/cloud/reference/release-notes/influx-cli.md (hunk 1 above)
Comment (kelseiv) [reply content/influxdb/cloud/reference/release-notes/influx-cli.md:43]: yes, thank you, @danxmoran

File: content/influxdb/cloud/reference/release-notes/influx-cli.md (hunk 2 above)
Comment (kelseiv) [reply content/influxdb/cloud/reference/release-notes/influx-cli.md:48]: Thank you, @danxmoran @sanderson - does adding the flag automatically compress as gzip, or do folks specify compression type? Is compression enabled or disabled by default?

Comment (danxmoran) [reply content/influxdb/cloud/reference/release-notes/influx-cli.md:48]: The flag takes an arg, either "none" or "gzip". The default is "gzip"

File: data/products.yml [hunk 7]
```diff
@@ -14,6 +14,9 @@ influxdb:
     "1.5": 5
     "1.4": 3
     "1.3": 9
+  latest_cli:
+    "2.0": 2.1.0
+    "2.1": foo
```
Comment (danxmoran) [inline data/products.yml:19]: Should this still be `foo`?

File: content/influxdb/v2.0/reference/cli/influx/backup/_index.md [hunk 8]
```diff
@@ -22,20 +22,21 @@ influx backup [flags] path
...
+|      | `--hide-headers`  | Hide table headers (default `false`)                                                                       |            | `INFLUX_HIDE_HEADERS` |
+|      | `--host`          | HTTP address of InfluxDB (default: `http://localhost:8086`)                                                | string     | `INFLUX_HOST`         |
+|      | `--json`          | Output data as JSON (default `false`)                                                                      |            | `INFLUX_OUTPUT_JSON`  |
+| `-o` | `--org`           | Organization name (mutually exclusive with `--org-id`)                                                     | string     | `INFLUX_ORG`          |
+|      | `--org-id`        | Organization ID (mutually exclusive with `--org`)                                                          | string     | `INFLUX_ORG_ID`       |
+|      | `--skip-verify`   | Skip TLS certificate verification                                                                          | string     |                       |
```
Comment (russorat) [inline content/influxdb/v2.0/reference/cli/influx/backup/_index.md:38]: this has a env var now

File: data/products.yml (hunk 7 above)
Comment (kelseiv) [reply data/products.yml:19]: @sanderson

File: content/influxdb/v2.0/reference/cli/influx/backup/_index.md (hunk 8 above)
Comment (kelseiv) [reply content/influxdb/v2.0/reference/cli/influx/backup/_index.md:38]: Yes, sorry about the confusion @russorat! this update is in another PR...I'm pulling in the other PR

**Review by russorat — APPROVED** [review]
LGTM

**Pull request merged**